src/javascript/EditPanel/EditPanel.utils.js

```diff
@@ -134,6 +134,10 @@ export function getDataToMutate({nodeData, formValues, sections, lang}) {
  * @returns {string} the name of the value property to use
  */
 export function getValuePropName(field) {
+    if (!field.multiple && field.selectorOptions?.find(option => option.name === 'password')) {
+        return 'encryptedValue';
+    }
+
```

Comment:
Please consider review the API, if we use a boolean it's not necessary to do that, instead we have to add a the boolean to encrypt the value in the mutation